packages/sanity/src/config/prepareConfig.ts

```diff
@@ -21,6 +22,7 @@ import {
   PreparedConfig,
   SingleWorkspace,
   WorkspaceOptions,
+  ConfigContext,
```

Comment:
Hm – what is `ConfigContext` in relation to `SourceContext`?

Reply:
It is the same thing, but SourceContext is a redefinition due to package direction. Called it source since everything is pulled from current source, but see preview comment.